microservice/frontend/catalog/debian/data/etc/nginx/sites-available/shop.microservice.io.1

```diff
@@ -0,0 +1,14 @@
+server {
+  listen   81; ## listen for ipv4; this line is default and implied
+  #listen   [::]:80 default ipv6only=on; ## listen for ipv6
+
+  root /usr/share/shop_A/frontend/catalog/html;
```

Comment:
This code is for the catalog microservice. This won't work, as the shop_A, shop_B site copy have been done on the cart microservice. See [this commit](https://github.com/devops-dojo/the-app/commit/51e23fd11650fe75700862cfb9a0e2d4765e40a7). 
Also, this is the cart micro service which will show the discount banner that we want to show in shop_A, shop_B.